.vscode/c_cpp_properties.json

```diff
@@ -6,16 +6,19 @@
                 "path": [
                     "${workspaceFolder}"
                 ],
-                "limitSymbolsToIncludedHeaders": true
+                "limitSymbolsToIncludedHeaders": false
             },
             "includePath": [
-                "${workspaceFolder}"
+                "~/paparazzi/sw/**",
```

Comment:
Maybe this path `~/paparazzi` is not guaranteed to be the same at every computer. But I like the fixes for VSCode!

Reply:
Same comment, just some junk to be removed :smile: